packages/CollapsibleText/stories/examples/Showcase.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
-import { Story, Rule, Tagline } from "storybook/assets/styles/common.styles";
+import { Story } from "storybook/assets/styles/common.styles";
 import { number, select } from "@storybook/addon-knobs";
-import Heading from "@paprika/heading";
+import StoryHeader from "storybook/components/StoryHeader";
```

Comment:
Nice, I like this! 👍 
While we're touching these stories, can we bump the Screener story into `Backyard/Tests/Screener`?